p2p/src/msg.rs

```diff
@@ -160,14 +152,14 @@ pub fn read_item<T: Readable>(stream: &mut dyn Read) -> Result<(T, u64), Error>
 /// until we have a result (or timeout).
 pub fn read_body<T: Readable>(h: &MsgHeader, stream: &mut dyn Read) -> Result<T, Error> {
 	let mut body = vec![0u8; h.msg_len as usize];
-	read_exact(stream, &mut body, time::Duration::from_secs(20), true)?;
+	stream.read_exact(&mut body)?;
```

Comment:
Similar question here, what's the biggest possible length for a `body`?  perhaps a Block is the biggest? then it's maximum possible value 1.5MB for example.

The timeout is 1 second and minimum speed is 1.5MB/s ?

Reply:
See my previous comment, the minimum speed is still 1 byte/sec